api/nodes/views.py

```diff
@@ -1920,7 +1925,7 @@ def postprocess_query_param(self, key, field_name, operation):
         if field_name == 'tags':
             if operation['value'] not in (list(), tuple()):
                 operation['source_field_name'] = 'tags__name'
-                operation['op'] = 'iexact'
+                operation['op'] = 'icontains'
```

Comment:
Why was this changed? I'm fairly certain that without changes to the model this will be egregiously slow, but benchmarking it would be a good idea. (There are [ways](https://docs.djangoproject.com/en/1.10/ref/contrib/postgres/search/) of making this faster, but the best way requires upgrading to Django 1.10)

Also, this seems out of scope. This ticket is for fixing node/contributor filtering, not node/tag filtering, and this isn't current prod behavior:

https://api.osf.io/v2/nodes/?filter[tags]=sci
https://api.osf.io/v2/nodes/?filter[tags]=science